server/enterprise/metrics/metrics.go

```diff
@@ -1244,6 +1249,42 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.ClientGlobalThreadsLoadDuration)
 
+	m.MobileClientLoadDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsMobileApp,
+			Name:      "mobile_load",
+			Help:      "Duration of the time taken from when a user opens the app and the app finally loads all relevant information (milliseconds)",
+			Buckets:   []float64{1, 1.5, 2, 3, 4, 4.5, 5, 5.5, 6, 7.5, 10, 20, 25, 30},
+		},
+		[]string{"platform", "agent"},
+	)
+	m.Registry.MustRegister(m.MobileClientLoadDuration)
+
+	m.MobileClientChannelSwitchDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsMobileApp,
+			Name:      "mobile_channel_switch",
+			Help:      "Duration of the time taken from when a user opens the app and the app finally loads all relevant information (milliseconds)",
```

Comment:
It seems like this description was accidentally copied from above

Reply:
I thought I updated these :facepalm: